include/mbedtls/pk.h

```diff
@@ -134,6 +134,12 @@ typedef struct mbedtls_pk_context {
     /* Public key information. */
     const mbedtls_pk_info_t *MBEDTLS_PRIVATE(pk_info);
 
+    /* The PSA key type of the key represented by the context.
+     *
+     * Note: Valid even if the PK object is not backed by a PSA key
+     * (as is the case for public keys). */
```

Comment:
Perhaps it's just me not reading it properly, but the way it's written now seem to suggest that there are other possibilities a part from public keys to get a PK context which is not backed by a PSA key, but this is not correct.
I would have rephrased it as:
```suggestion
     * Note: Valid even for public keys which are not backed by a PSA key */
```

This is really minor though and perhaps I overthought about it, so if you think this is not important feel free to ignore it :)

Reply:
I've reworded it with a comma for extra clarity 🙂